graspologic/embed/ase.py

```diff
@@ -158,4 +167,64 @@ def fit(self, graph, y=None):
 
         self.n_features_in_ = len(A)
         self._reduce_dim(A)
+
+        # for out-of-sample
+        inv_eigs = np.diag(1 / self.singular_values_)
+        self._pinv_left = self.latent_left_ @ inv_eigs
+        if self.latent_right_ is not None:
+            self._pinv_right = self.latent_right_ @ inv_eigs
+
+        self.is_fitted_ = True
         return self
+
+    def transform(self, X):
+        """
+        Obtain latent positions from an adjacency matrix or matrix of out-of-sample vertices.
+        For more details on transforming out-of-sample vertices, see [2].
```

Comment:
maybe sphinx reference the tutorial rather than the article.

Reply:
da4780c